src/vmm/src/builder.rs

```diff
@@ -674,7 +731,7 @@ pub fn build_microvm(
     if let Some(vsock) = vm_resources.vsock.get() {
         attach_unixsock_vsock_device(&mut vmm, vsock, event_manager, intc.clone())?;
         #[cfg(not(feature = "net"))]
-        vmm.kernel_cmdline.insert_str("tsi_hijack")?;
+        //vmm.kernel_cmdline.insert_str("tsi_hijack")?;
         #[cfg(feature = "net")]
```

Comment:
Should this be CCA only? There's another one a few lines below as well

Reply:
I can't remember why I commented this line.